nautobot/docs/docker/index.md

```diff
@@ -0,0 +1,159 @@
+# Nautobot Docker Images
+
+Nautobot is packaged as a Docker image for use in a production environment. The published image is based on the `python:3.6-slim` image to maintain the most compatibility with Nautobot deployments. The Docker image and deployment strategies are being actively developed, check back here or join the **#nautobot** Slack channel on [Network to Code](https://networktocode.slack.com) for the most up to date information.
+
+## Tags
+
+The Docker image is published to both the GitHub container registry as well as Docker Hub.
+
+To get the image from Docker Hub run:
+
+```no-highlight
+docker image pull networktocode/nautobot
+```
+
+!!! info
+    Effective November 20, 2020, [rate limits](https://www.docker.com/increase-rate-limits) have gone into effect on Docker Hub.  To increase the rate limit you can use `docker login` to authenticate with an account, however using the GitHub container registry avoids these limits all together.
+
+To get the image from the GitHub Container Registry (GHCR) run:
+
+```no-highlight
+$ docker image pull ghcr.io/nautobot/nautobot
+```
+
+The following tags are available:
+
+* `X.Y.Z` these images are built with the same baseline as the released Python packages based on the default python version (3.6) docker container
+* `latest` these images are built from the latest code in the main branch (should be the latest released version) based on the default python version (3.6) docker container
+* `X.Y.Z-py${PYTHON_VER}` these images are built with the same baseline as the released Python packages based on the python version ($PYTHON_VER) docker container
+* `latest-py${PYTHON_VER}` these images are built from the latest code in the main branch (should be the latest released version) based on the python version ($PYTHON_VER) docker container
+* `develop` these images are built from the latest code in the develop branch on each commit based on the default python version (3.6) docker container
+* `develop-${GIT_SHA:0:7}-$(date +%s)` tags for each commit to the develop branch based on the default python version (3.6) docker container
+* `develop-latest-py${PYTHON_VER}` these images are built from the latest code in the develop branch on each commit based on the python version ($PYTHON_VER) docker container
```

Comment:
In trying to integrate my plugin's CI with these tagged images, I find this one a bit inconvenient:

- Nautobot `1.0.0`, Python `3.6` --> `1.0.0-py3.6`
- Nautobot `1.0.0`, Python `3.7` --> `1.0.0b3-py37`
- Nautobot `develop`, Python `3.8` --> `develop-latest-py38` (have to special-case the `-latest` here)

Would it make sense to change this one to simply `develop-py{PYTHON_VER}`, or is this just something I'll need to special-case in my CI code?

Reply:
@glennmatthews sorry, someone else pointed this out as well, this doc is out of date `develop-py3.8` is available.